Projects/ChangeTier/Sources/Navigation/ChangeTierNavigation.swift

```diff
@@ -0,0 +1,88 @@
+import Foundation
+import SwiftUI
+import hCore
+import hCoreUI
+import hGraphQL
+
+public class ChangeTierNavigationViewModel: ObservableObject {
+    @Published public var isEditTierPresented = false
+    @Published public var isEditDeductiblePresented = false
+    @Published public var isCompareTiersPresented = false
+    @Published public var isInsurableLimitPresented: InsurableLimits?
+    @Published public var isTierLockedInfoViewPresented = false
+
+    @StateObject var router = Router()
+    var vm = ChangeTierViewModel()
+
+    init() {}
+}
+
+public enum ChangeTierSource {
+    case termination
```

Comment:
I think we should add here multiple options for the termination:
It is expected to have this sources as an input for the underwriter source: - we don't have to call them like that but like our internal types.
eg
```
public enum ChangeTierSource {
    case changeTier
    case betterPrice
    case betterCoverage
}
```
and later on map to what is expected from us to the underwriter type
```
enum ChangeTierDeductibleSource {
    SELF_SERVICE,
    TERMINATION_BETTER_PRICE,
    TERMINATION_BETTER_COVERAGE,
}
```